.github/workflows/release.yaml

```diff
@@ -0,0 +1,81 @@
+name: "Release"
+
+on:
+  push:
+    tags:
+      - "**"
+
+jobs:
+  release:
+    name: "Release"
+
+    runs-on: "ubuntu-latest"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.3"
+
+    steps:
+      - name: "Checkout"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          php-version: "${{ matrix.php-version }}"
+          ini-values: memory_limit=-1
+          tools: composer:v1, cs2pr
+
+      - name: "Get composer cache directory"
+        id: composercache
+        run: echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: "Cache dependencies"
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.composercache.outputs.dir }}
+          key: ${{ runner.os }}-php-${{ matrix.php-version }}-${{ matrix.dependencies }}-composer-${{ hashFiles('**/composer.json') }}
+          restore-keys: ${{ runner.os }}-php-${{ matrix.php-version }}-${{ matrix.dependencies }}-composer-
+
+      - name: "Install dependencies"
+        run: "composer update --no-interaction --no-progress"
+
+      - name: "Download BOX.phar"
+        run: "wget https://github.com/humbug/box/releases/latest/download/box.phar"
+
+      - name: "Validate configuration for humbug/box"
+        run: "php box.phar validate"
+
+      - name: "Compile paratest.phar with humbug/box"
+        run: "php box.phar compile"
+
+      - name: "Show info about paratest.phar with humbug/box"
+        run: "php box.phar info bin/paratest.phar"
+
+      - name: "Determine tag"
+        id: "determine-tag"
+        run: "echo \"::set-output name=tag::${GITHUB_REF#refs/tags/}\""
+
+      - name: "Create release"
```

Comment:
@andreasschroth this morning I've tested the Release Github Actions workflow.

The first time I've tried to tag the release from the Github GUI page https://github.com/paratestphp/paratest/releases/new but the GA workflow try to create that release itself and I got the error you can see at https://github.com/paratestphp/paratest/runs/917774095#step:12:9

So I did a bare `git tag` locally and pushed it, and everything worked fine with the expected PHAR attached:

1. https://github.com/paratestphp/paratest/runs/917835737
1. https://github.com/paratestphp/paratest/releases/tag/4.1.1-alpha.2

You'll see in fact the releaser is @github-actions

I don't like it very much because we can't create a release with the Github GUI anymore. The solutions may be:

1. Accept it as a new release process; if we want to add a description to the release, we'll edit the one created by GA
1. Get rid of `actions/create-release` within GA, create the release only with Github GUI so `actions/upload-release-asset` can play as expected manually providing the `upload_url`

WDYT?

Reply:
A third and elegant solution is to use https://github.com/laminas/automatic-releases

Everytime a milestone is closed, the release is automatically created with the list of the issue/pr closed within it.